### ISSUE TITLE
Baker crafts golden breads even without a request

### ISSUE THREAD

**Issue by pantsguyapp** [ldtteam/minecolonies#7804]
### Prerequisites 

- [X] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [X] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [X] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version:
- MineColonies Version:
- Structurize Version:
- Related Mods and their Versions:

## Expected behavior

The baker should probably not use the precious golds to craft some breads on their own initiative and wait for a request for that

## Actual behavior

They craft the golden breads for the restaurant, wasting all the gold

## Steps to reproduce the issue

1. Put some gold ingots in the warehouse
2. Wait for sometimes and check the restaurant

### Logs

- latest.log: 
- crashlog: 

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Raycoms** [comment]
Did you try turning off golden bread as food in the restaurant?

**Comment by pantsguyapp** [comment]
> Did you try turning off golden bread as food in the restaurant?

No, because I thought they would not waste my resources just like that.

**Comment by Raycoms** [comment]
Well, wasting resources is relative. One mans junk is another mans treasure. The cook requests everything that's enabled. And the baker produces everything that's requested. When you're warehouse is out of food, the baker will take over.

**Comment by pantsguyapp** [comment]
The thing is that I was completely unaware of the golden bread, I was having the baker to auto-feed my restaurant with normal breads using the minimum stock function, I suggest that when I upgraded to Level 5 Baker it automatically switch to make the golden breads. This could be hazardous, people might be unaware of this and have all of their colony's gold be turned into breads, there should be a warning or something to search for the golden breads manually in the restaurant and turn it off.

"One mans junk is another mans treasure"? I think the one who considers gold "junk" are the colonists as they can't do much with it at all, I am the one with a mesuarable value for golds, and I would appreciate it that there is some precaution to me about the baker consume all my golds for this foods, furthremore, the fact that colonists get little benefit when consuming this bread angers me, not even the 2 hearts healing, as they would do to enchanted golden apples.

In my case, I ordered 1 golden apple from the postbox for a Uni research, , I then taught the recipe to the cook then deposited 8 ingots so that the colonists have something to work with and that is when I discovered that all the golds got turned into breads instead, meaning the bread recipe was prioritied over my request!!!.
